core/iwasm/aot/arch/aot_reloc_xtensa.c

```diff
@@ -219,6 +222,9 @@ apply_relocation(AOTModule *module, uint8 *target_section_addr,
             initial_addend = *(int32 *)insn_addr;
             *(uintptr_t *)insn_addr = (uintptr_t)symbol_addr + initial_addend
                                       + (intptr_t)reloc_addend;
+#if (WASM_MEM_DUAL_BUS_MIRROR != 0)
+            os_dcache_flush();
```

Comment:
Don't need to flush cache in each relocation, when this done, `load_from_sections` will do this when loading sections is done.

Reply:
How about adding some comments in this PR? developer may be confused about the code.